src/shared/components/home/site-form.tsx

```diff
@@ -159,6 +160,9 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
         blocked_instances: this.props.siteRes.federated_instances.andThen(
           f => f.blocked
         ),
+        taglines: Some(
```

Comment:
Why is Some used here, since tagline isn't required and might be missing?

Reply:
I get it, because an empty string *sort of* denotes missing data. But to me, its much more logically consistent to use `Option` when a value is missing, then front ends don't have to do array length checks specifically for array types, as compared to other types. 

I went through the process recently of replacing all javascript nullpointerexception possibilities with `Option` monads, and it eliminated a whole class of compile-time bugs.